cirq-superstaq/cirq_superstaq/daily_integration_test.py

```diff
@@ -303,7 +303,10 @@ def test_submit_to_provider_simulators(target: str, service: css.Service) -> Non
     assert job.counts(0) == {"11": 1}
 
 
-@pytest.mark.skip(reason="Can't be executed when Hilbert is set to not accept jobs")
+@pytest.mark.skipif(
+    "cq_hilbert_qpu" not in css.Service().get_targets()["compile-and-run"],
+    reason="Can't be executed when Hilbert is set to not accept jobs",
+)
```

Comment:
Post `v0.5.0`, (see https://github.com/Infleqtion/client-superstaq/pull/788 and https://github.com/Infleqtion/server-superstaq/pull/2822) `get_targets()` no longer returns a dictionary and hence doesn't have a `["compile-and-run"]` anymore. You would have to do something like: 

```ruby
cq_target = gss.typing.Target(
        target="cq_hilbert_qpu",
        supports_submit=False,
        supports_submit_qubo=False,
        supports_compile=True,
        available=True,
        retired=False,
    )
cq_target not in css.Service().get_targets()
```
or maybe: 

```ruby
non_submit_targets = [Target.target for Target in css.Service().get_targets(supports_submit=False)]
"cq_hilbert_qpu" in non_submit_targets
```